### PR TITLE
uncommented prerel section + removed iframe api from stable

### PR DESCRIPTION
**Pull request by jasonstephen15**
**Revised section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1680#10107254
and un-commented the "1.0.1133-prerelease" section below it.

Writer/Editor pass: vchapel

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -103,22 +78,22 @@ The following APIs are promoted to stable in this prerelease SDK:
...
-
+*  The ProcessInfo APIs provides more information about WebView2 [processes](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprocessinfo?view=webview2-1.0.1133-prerelease&preserve-view=true) and [process collections](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprocessinfocollection?view=webview2-1.0.1133-prerelease&preserve-view=true).
 *  The [HTTP Authentication API](/microsoft-edge/webview2/reference/win32/icorewebview2experimental10?view=webview2-1.0.1133-prerelease&preserve-view=true).
 
 #### Bug Fixes
   
-*   Fixes a bug that prevented Set-Cookies header from showing up in the `WebResourceResponseReceived` event.  
-*   Resolved a bug where pop-ups and owned windows would jump to a different position before closing instead of closing along with the app window. This bug was only active for a very short window of time.  
+*   Fixed a bug that prevented Set-Cookies header from showing up in the `WebResourceResponseReceived` event.
```
Comment (vchapel) [inline microsoft-edge/webview2/release-notes.md:86]:
```suggestion
*   Fixed a bug that prevented `Set-Cookies` header from showing up in the `WebResourceResponseReceived` event.  
```

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -103,22 +78,22 @@ The following APIs are promoted to stable in this prerelease SDK:
...
+*   Fixed a bug that prevented Set-Cookies header from showing up in the `WebResourceResponseReceived` event.  
+*   Resolved a bug where pop-ups and owned windows would jump to a different position before closing instead of closing 
+along with the app window. This bug was only active for a very short window of time.  
 *   Fixed focus issue after closing file picker dialog.  
 *   Fixed bug where Find on Page UI visibility did not change with WebView2 visibility.  
-*   Fixed bug where `GetAvailableBrowserVersionString()` fails to locate/load 'WebView2Loader.dll'([Issue #1236](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1236))
-*   Fixed size and position of the new window created with window.open when `NewWindowRequested` event was not 
-handled.([Issue #1343](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1343))
+*   Fixed bug where `GetAvailableBrowserVersionString()` fails to locate/load `WebView2Loader.dll` ([Issue #1236](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1236))
+*   Fixed size and position of the new window created with window.open when NewWindowRequested event was not
```
Comment (vchapel) [inline microsoft-edge/webview2/release-notes.md:92]:
```suggestion
*   Fixed size and position of the new window created with `window.open` when `NewWindowRequested` event was not 
```

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -31,10 +31,9 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, go to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 <!-- ====================================================================== -->
-
 ## 1.0.1072.54
   
-Release Date: January 13, 2022
+Release Date: January 13, 2022  
   
 [NuGet package for WebView2 SDK 1.0.1072.54](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1072.54)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:39]: "WebView2 Runtime version 97.1072.54"
Supposed to be 4-part instead of 3-part version #?

Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:39]: "requires WebView2 Runtime version 95.1020.30"
3-part #?  Supposed to be 4-part?

**Review by vchapel — APPROVED** [review]
LGTM

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -31,94 +31,69 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
-  
-Release Date: January 13, 2022  
-  
-[NuGet package for WebView2 SDK 1.0.1133](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1133-prerelease)  
-  
-For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 99.0.1133.0 or higher.  
+
+Release Date: January 13, 2022
+
+[NuGet package for WebView2 SDK 1.0.1133](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1133-prerelease)
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:59]:
```suggestion
[NuGet package for WebView2 SDK 1.0.1133-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1133-prerelease)
```

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -31,94 +31,69 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
-*   Fixed bug where `GetAvailableBrowserVersionString()` fails to locate/load 'WebView2Loader.dll'([Issue #1236](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1236))
-*   Fixed size and position of the new window created with window.open when `NewWindowRequested` event was not 
-handled.([Issue #1343](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1343))
+
+*   Fixed a bug that prevented `Set-Cookies` header from showing up in the `WebResourceResponseReceived` event.
+*   Resolved a bug where pop-ups and owned windows would jump to a different position before closing instead of closing
+along with the app window. This bug was only active for a very short window of time.
+*   Fixed focus issue after closing file picker dialog.
+*   Fixed bug where Find on Page UI visibility did not change with WebView2 visibility.
+*   Fixed bug where `GetAvailableBrowserVersionString()` fails to locate/load `WebView2Loader.dll` ([Issue #1236](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1236))
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:91]:
```suggestion
*   Fixed bug where `GetAvailableBrowserVersionString()` fails to locate/load `WebView2Loader.dll`. ([Issue #1236](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1236))
```

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -31,94 +31,69 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+*   Fixed size and position of the new window created with `window.open` when `NewWindowRequested` event was not
+handled. ([Issue #1343](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1343))
 *   Fixed bug where mini menu was still displaying on selected text when context menus were disabled. This change is Runtime-specific.([Issue #1345](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1345))
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:94]:
```suggestion
*   Fixed bug where mini menu was still displaying on selected text when context menus were disabled. This change is Runtime-specific. ([Issue #1345](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1345))
```

**Review by peiche-jessica — APPROVED** [review]
Approved with minor nitpicking.

**Pull request merged**